upcloud/service/service.go

```diff
@@ -94,23 +88,6 @@ func New(client *client.Client) *Service {
 	return &service
 }
 
-// GetAccount returns the current user's account
-func (s *Service) GetAccount() (*upcloud.Account, error) {
-	account := upcloud.Account{}
-	response, err := s.basicGetRequest("/account")
-
-	if err != nil {
-		return nil, err
-	}
-
-	err = json.Unmarshal(response, &account)
-	if err != nil {
-		return nil, err
-	}
-
-	return &account, nil
-}
```

Comment:
Hmm, is removing this a breaking change? 🤔  If we don't want to bump the library that much, we could also leave this in and maybe call the account service here

Reply:
I just moved those account related things to account.go file. It's in same package so there shouldn't be any visible changes to users.

Reply:
Ok, cheers! I'm not super familiar with Go so I thought you could somehow import this service directly and call `service.GetAccount()` which I was guessing could break 👍